packages/atomic/src/pages/index.html

```diff
@@ -3,50 +3,26 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
-    <title>Atomic Components</title>
+    <title>Coveo Atomic</title>
 
     <script type="module" src="/build/atomic.esm.js"></script>
     <script nomodule src="/build/atomic.js"></script>
     <link rel="stylesheet" href="/themes/coveo.css" />
+    <link rel="stylesheet" href="https://use.typekit.net/bqa0xml.css" />
```

Comment:
Is it "okay" to reference this for our official coveo theme ? 🤔

Reply:
Yes Gibson is part of the Coveo theme. The default theme will use system fonts and colors with more contrast

Reply:
My question was mostly about:  Should we ship the font as part of our css files, and not reference use.typekit.net ?

Reply:
Ah I see, alright that's a good idea I'll add it in the bundle